### ISSUE TITLE
replicate problem, train process

### ISSUE THREAD

**Issue by ruixu2** [lululxvi/deep-learning-for-indentation#13]
Prof Lu, hello!
    I try to replicatie mfnn with pytorch. However, i am not sure how to train it. 

In the mfnn without residual in the ref.34， the train process is training the low fidelity model sperately (only update the low model params), then training the high fidelity model (update two part parameters).  For example，20000 epochs for low model，then 20000 epochs for high model.

In this work, it seems training simultaneously（if low fidelity data, only update low model; if high fidelity data， update two model parameters）in a epoch. 
![deepxde 1.1.12 source code](https://user-images.githubusercontent.com/34615161/173726527-db894272-cc03-44bf-9320-3947cbdbe718.png)

In the E* prediction，only the low model（[3, 128, 128, 1]），it can get good result with mape < 5%, But when I train high fidelity model, on the contrary，it get poor result.
![image](https://user-images.githubusercontent.com/34615161/173727043-007a2585-eab0-4ae1-87e3-c4cd1e1a6919.png)

In the sigma y prediction, it has no convergence trend.

Another problem is the mfnn with residual, 
$$y_{H}=a_{L} y_{L}+\epsilon(tanh α_{1} . f_{linear}(x, y_L)+tanh α_{2} . f_{nonlinear}(x,y_L))$$

 Is it  can be replacedby  a linear+tanh with input [linear part, nonlinear part] ?
(I can not understand how to train the two alpha parameters)

**Comment by ruixu2** [comment]
sigma_y convergenced! it must do normalization!

**Comment by ruixu2** [comment]
Thanks!
